src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.spec.ts

```diff
@@ -339,9 +342,9 @@ describe('HealthComponent', () => {
       expect(component['calcPercentage'](1, 0)).toEqual(0);
       expect(component['calcPercentage'](undefined, 1)).toEqual(0);
       expect(component['calcPercentage'](null, 1)).toEqual(0);
-      expect(component['calcPercentage'](0, 1)).toEqual(0);
-      expect(component['calcPercentage'](2.346, 10)).toEqual(23);
-      expect(component['calcPercentage'](2.35, 10)).toEqual(24);
+      expect(component['calcPercentage'](0, 1)).toEqual('0.00');
```

Comment:
What would happen if there's 1 degraded object out of 100,000? Would we expect to see a 0.01% or a 0%?

Reply:
It would show the result up to 2 decimal places, so the result would be `0.00%`. 1 degraded object out of 10,000 would give `0.01%`

Reply:
So in that case, since I asked 100,000, it would give you 0.00%. So I guess this doesn't fix the issue (users shouldn't see something as 100% ok when there's actually a 0,000001% thing wrong). So I guess you should use the ceiling operator here, don't you think?

Reply:
Yes, for this we can either show the exact value without rounding up or ceiling it, or we can use ceil operator to show the upper limit. I have modified it to ceil now. Thanks